sprint/tutorials/environment_setup.md

```diff
@@ -0,0 +1,191 @@
+
+(environment_setup)=
+# Environment setup
+
+## Pre-requisites
+
+### GitHub account
+To contribute you'll need a GitHub account. You can create one from [this link](https://github.com/join)
+
+::::{sidebar} New to Git and GitHub?
+We have tried to cover all the steps and include both input and output, to make
+sure you can follow this tutorial without knowing Git.
+However, if you want to learn about Git and understand what is going on,
+we recommend [this dataschool tutorial](https://www.dataschool.io/how-to-contribute-on-github/)
+::::
+
+### Install Git
+
+::::{tab-set}
+:::{tab-item} Linux
+:sync: linux
+Instructions with installation commands for most Linux distributions
+can be found on the [Git project website](https://git-scm.com/download/linux)
+:::
+:::{tab-item} macosx
+:sync: macos
+Instructions with installation commands for macosx
+can be found on the [Git project
+website](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git#_installing_on_macos)
+:::
+:::{tab-item} Windows
+:sync: windows
+The recommended way to install Git is using [Git for Windows](https://git-scm.com/download/win).
+:::
+::::
+
+### Install conda
+We recommend you install miniconda and follow the steps below to install only
+the libraries you'll need.
+
+::::{tab-set}
+:::{tab-item} Linux
+:sync: linux
+Miniconda installation instructions for linux are available on the [conda website](https://conda.io/projects/conda/en/latest/user-guide/install/linux.html)
+:::
+:::{tab-item} macosx
+:sync: macos
+Miniconda installation instructions for macosx are available on the [conda website](https://conda.io/projects/conda/en/latest/user-guide/install/macos.html)
+:::
+:::{tab-item} Windows
+:sync: windows
+Miniconda installation instructions for windows are available on the [conda website](https://conda.io/projects/conda/en/latest/user-guide/install/windows.html)
+:::
+::::
+
+### Text editor
+You might already have a text editor installed. Check if you have any editor from
+the list below, otherwise install _one_, any will do:
+
+* [Atom](https://atom.io/)
+* [Visual Studio Code](https://code.visualstudio.com/)
+* [Sublime text](https://www.sublimetext.com/)
+* [PyCharm (community)](https://www.jetbrains.com/pycharm/download)
+
+## Fork the pymc repo
+Fork the [project repository](https://github.com/pymc-devs/pymc/)
+by clicking on the 'Fork' button near the top right of the main repository page.
+
+![fork_button](https://docs.github.com/assets/cb-6294/images/help/repository/fork_button.jpg)
```

Comment:
This image is not working anymore. We should download it and add it to the images folder. It now seems to be at https://docs.github.com/assets/cb-28613/images/help/repository/fork_button.png